types/rdfjs__fetch-lite/index.d.ts

```diff
@@ -1,9 +1,10 @@
 import { Formats } from "@rdfjs/formats";
 import { BaseQuad, DatasetCore, DatasetCoreFactory, Quad, Stream } from "@rdfjs/types";
 
-export interface FormatsInit extends RequestInit {
-    formats: Formats;
+export interface FormatsInit extends Omit<RequestInit, "body"> {
+    formats?: Formats;
```

Comment:
Contrary to the packages's readme, `formats` are not required directly but can be provided by the RDF/JS environment: https://github.com/rdfjs-base/fetch-lite/blob/master/Factory.js#L9

Reply:
> Contrary to the packages's readme, formats are not required directly but can be provided by the RDF/JS environment: https://github.com/rdfjs-base/fetch-lite/blob/master/Factory.js#L9

Can you PR a change to the readme as well - happy to approve this one @bergos signs off on the change in the docs.

Reply:
I cannot. I'm banned from all Thomas' repositories